### DISCUSSION TITLE
Tag Field is cleared on selection, but dropdown still show previous results.

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#5008] [Ideas & Feature Requests] [answered]
- Nova Version: 4.18.1

### Description:
When you click on a tag field, the dropdown only open when you start typing. (ok)
Selection a value from the dropdown, the search field is cleared and the focus is lost. (ok)
Focus the search field again to search for something else, the dropdown open and display the previous results.

![2022-11-02 15 48 35](https://user-images.githubusercontent.com/17392251/199588940-71da902c-f82d-492b-bcd5-f035c88ddedd.gif)

**Comment by crynobone** [comment]
This behavior is consistent with `Select`/`BelongsTo`/`MorphTo` using `searchable()`. Marking as feature request.

**Comment by davidhemphill** [comment, marked as answer]
This is an intentional decision as it allows one to specify multiple tags from the same search. 

Example: I type in 'D' and can tag a Post with both 'Design' and 'Development.'

**Reply by mrleblanc101** [reply]
Ok, i guess it's strange because it close the dropdown on selection and clear the search. Vue-multiselect also work like this but it's can be customized, so that the dropdown stay open and the search doesn't get cleared.

**Reply by davidhemphill** [reply]
We could look into doing something like that. We didn't because when set to tag view, you can't actually see what is added to the tag list because the dropdown covers it.

**Reply by mrleblanc101** [reply]
But if you keep the dropdown open, you could display it in the list with a checkmark or an highlight.
Maybe this could be an option `->closeOnSelection()`
